microcosm_pubsub/handlers/uri_handler.py

```diff
@@ -132,15 +140,24 @@ def get_resource(self, message, uri):
         Passes message context.
 
         """
+        if self.resource_cache:
```

Comment:
will only evaluate to truth-y value if "resource_cache" is included in graph.use(...) *and* graph.config.resource_cache.enabled is set to `True`